tests/openkit-shared-tests/Core/Communication/BeaconSendingCaptureOnStateTest.cs

```diff
@@ -228,7 +271,7 @@ public void MultiplicityIsSetToZeroIfNoFurtherNewSessionRequestsAreAllowed()
             // also ensure that both got a configuration set
             Assert.That(sessionOne.IsBeaconConfigurationSet, Is.True);
             Assert.That(sessionOne.BeaconConfiguration.Multiplicity, Is.EqualTo(0));
-            
+

```

Comment:
remove this unnecessariliy introduced newline

Reply:
Was not really introduced, but was there already (document reformat).

Thanks to your comment, I saw an old issue in the test. :)